test/test.js

```diff
@@ -473,6 +491,23 @@ describe('YeldaChat', () => {
   })
 
   describe('yeldaChat.resetChat', () => {
+
+    it('should load data of the second call', async () => {
```

Comment:
```suggestion
    it('should set globally the params data from the latest call, async () => {
```

would be closer to reality : this test does not check that the latest (better than second) call data have been loaded)